app/lib/server/server.go

```diff
@@ -144,10 +146,17 @@ func InteractiveConfig(binaryName string) {
 
 	// create the keys
 	privStr, pubStr := createKeyPair()
+	ips, err := net.LookupIP(publicAddress.Host())
+	log.ErrFatal(err)
+	cert, key, err := network.NewCertKey(network.NewTLSCert(big.NewInt(1), "ch", "epfl", "dedis",
```

Comment:
I'll never hate you, or only for a short time - Love is the source of all live ;)

Do you think it's better to add it as arguments to `setup` or add more user-input?

With regard to already existing PEM encoded certificate + key - where would those come from? Apache/Postfix? Letsencrypt?